kube/src/api/subresource.rs

```diff
@@ -258,36 +258,6 @@ impl Default for AttachParams {
 
 #[cfg(feature = "ws")]
 impl AttachParams {
-    /// Specify the container to execute in.
-    pub fn container<T: Into<String>>(&mut self, container: T) -> &mut Self {
-        self.container = Some(container.into());
-        self
-    }
-
-    /// Set `stdin` field.
-    pub fn stdin(&mut self, enable: bool) -> &mut Self {
-        self.stdin = enable;
-        self
-    }
-
-    /// Set `stdout` field.
-    pub fn stdout(&mut self, enable: bool) -> &mut Self {
-        self.stdout = enable;
-        self
-    }
-
-    /// Set `stderr` field.
-    pub fn stderr(&mut self, enable: bool) -> &mut Self {
-        self.stderr = enable;
-        self
-    }
```

Comment:
i think these are still good to include, matches the builder pattern in our other resource params